projects/go-lib/src/lib/components/go-tab/go-tab.module.ts

```diff
@@ -0,0 +1,26 @@
+import { NgModule } from '@angular/core';
+
+import { GoTabComponent } from './go-tab.component';
+import { CommonModule } from '@angular/common';
+import { GoTabGroupComponent } from './go-tab-group.component';
+import { GoButtonModule } from '../go-button/go-button.module';
+import { GoCardModule } from '../go-card/go-card.module';
+
+
+@NgModule({
+  declarations: [
+    GoTabComponent,
+    GoTabGroupComponent
+  ],
+  imports: [
+    CommonModule,
+    GoButtonModule,
```

Comment:
It doesn't seem like we need to import `GoButtonModule` or `GoCardModule` here. Do we?

Reply:
Nope. I was originally using them and then the styles weren't working so I removed them